rsyslog/map.jinja

```diff
@@ -152,3 +152,4 @@ RedHat:
         enabled: true
 {%- endload %}
 {%- set global = salt['grains.filter_by'](base_defaults, merge=salt['pillar.get']('rsyslog:client')) %}
```

Comment:
I agree, but it is out of scope of this commit, decision to merge client settings into global settings, was made long before this commit. I understand that it should be separated, but it will require much more time and effort.